packages/components/src/local/molecules/json-editor/index.tsx

```diff
@@ -74,19 +73,25 @@ export const JsonEditor: React.FC<Props> = ({
     return memoryName
   }
 
+  const getEditLocalStorgaeid = (): string => `edit-${genLocalStorageId()}`
```

Comment:
@lilitkarapetyan - we have two methods with similar names (`getEditLocalStorageId` and `genLocalStorageId`).  Can you put in comments to indicate what they are both for, please?

Reply:
Ok I will add a comment in the code.